pertpy/plot/_guide_rna.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+import logging
+
+import numpy as np
+import scanpy as sc
+from anndata import AnnData
+
+
+class GuideRnaPlot:
```

Comment:
Maybe rename this to "GuideAssignmentPlot" or something?

Reply:
It is actually not about guide assignment only. We use it to have a good visualization before and after assignment.
It is a more like heatmap ordered by guide expressions so that we see beautiful groups. Alternatively, a user can provide the ordering or get the ordering for future use.